src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/csip/CSIPService.kt

```diff
@@ -321,7 +322,7 @@ class CSIPService(
             closeDate = request.closeDate,
             recordedDate = request.recordedDate,
             recordedUser = request.recordedBy,
-            reviewSequence = reviewRepository.getNextSequence(this.id),
+            reviewSequence = request.reviewSequence,
```

Comment:
The sequence is not part of the primary key - it is just to order the list of reviews